app-modules/meeting-center/src/Managers/GoogleCalendarManager.php

```diff
@@ -12,10 +12,26 @@
 use Google\Service\Calendar\EventDateTime;
 use Assist\MeetingCenter\Models\CalendarEvent;
 use Google\Service\Calendar as GoogleCalendar;
+use Google\Service\Calendar\CalendarListEntry;
 use Assist\MeetingCenter\Managers\Contracts\CalendarInterface;
 
 class GoogleCalendarManager implements CalendarInterface
 {
+    /**
+     * @return array<string, string>
+     */
+    public function getCalendars(Calendar $calendar): array
+    {
+        $service = (new GoogleCalendar($this->client($calendar)));
+
+        return collect($service->calendarList->listCalendarList()
+            ->getItems())
+            ->filter(fn (CalendarListEntry $item) => ! str($item->id)->endsWith('@group.v.calendar.google.com'))
```

Comment:
Just curios, what is the purpose of the filter here? Are there other items in here that we would not want?

Reply:
There are 2 default calendars I have and most probably have: Holidays in United States, and Birthdays. They share this partial id `@group.v.calendar.google.com` which is different from the partial id of a calendar you create manually `@group.calendar.google.com` so I'm filtering those out. I couldn't find a better way because the other data on each of those appears like a user created calendar.